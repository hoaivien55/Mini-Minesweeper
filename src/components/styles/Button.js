import styled from "styled-components";

const Button = styled.button`
	min-width: 120px;
	background: #65dfa1;
	border: none;
	height: 30px;
	border-radius: 12px;
	color: white;
	font-weight: bold;
	text-transform: uppercase;
    cursor: pointer;
    &:hover {
        opacity: 0.8;
    }
`;

export { Button };
