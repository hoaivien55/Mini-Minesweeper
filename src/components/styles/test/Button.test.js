import renderer from 'react-test-renderer';
import { Button } from '../Button';

it('Button: Virtual DOM snapshot', () => {
    const tree = renderer
      .create(<Button />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
