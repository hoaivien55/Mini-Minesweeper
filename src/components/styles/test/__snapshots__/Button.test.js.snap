// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button: Virtual DOM snapshot 1`] = `
.c0 {
  min-width: 120px;
  background: #65dfa1;
  border: none;
  height: 30px;
  border-radius: 12px;
  color: white;
  font-weight: bold;
  text-transform: uppercase;
  cursor: pointer;
}

.c0:hover {
  opacity: 0.8;
}

<button
  className="c0"
/>
`;
